src/renderer/components/deposit/add/PendingAssetsWarning.tsx

```diff
@@ -22,16 +22,16 @@ const AssetIconAmount: React.FC<AssetIconAmountProps> = (props): JSX.Element =>
     loading
```

Comment:
Just moving existing content of `Deposit.subcomponents` into `PendingAssetsWarning` and using renamed style names - changes can be ignored